packages/unrar.rb

```diff
@@ -3,24 +3,24 @@
 class Unrar < Package
   description 'UnRAR is a powerful archive extractor.'
   homepage 'http://www.rarlab.com/'
-  version '5.6.3'
+  version '7.0.6'
   license 'unRAR'
   compatibility 'all'
-  source_url 'https://www.rarlab.com/rar/unrarsrc-5.6.3.tar.gz'
-  source_sha256 'c590e70a745d840ae9b9f05ba6c449438838c8280d76ce796a26b3fcd0a1972e'
-  binary_compression 'tar.xz'
+  source_url 'https://www.rarlab.com/rar/unrarsrc-7.0.6.tar.gz'
+  source_sha256 'd9c0d1c881603aa2fa38248634bf9c9d63a6a3bdb5fa17dc9358ebf8b61f0cd2'
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'ebf07529e4586d9494a41e0c0bc08d9c45b706a953ed9dc224b93e67f6b26cab',
-     armv7l: 'ebf07529e4586d9494a41e0c0bc08d9c45b706a953ed9dc224b93e67f6b26cab',
-       i686: '352da95a4126086f58fb69ae4aa4301f5464d52ac7c0d527e0434aaa6c846bee',
-     x86_64: 'ff5bc1b1c1b2f04cfc69b42345df16cac2bfc4be64a15016825e7980b4f5683f'
+    aarch64: '9f3f6421b5570cb695f841cce31b47b7585ea351e48d43844c12466700c88a21',
+     armv7l: '9f3f6421b5570cb695f841cce31b47b7585ea351e48d43844c12466700c88a21',
+       i686: '71e544df5bf2ada5f716cff0d516c81c63e883eb7dac07bb1fa5c371026fb44e',
+     x86_64: '911088681e176d910c5401c82b4982f43df1541e97c9d54761f318c96ae76cf7'
   })
 
   def self.build
     # force to compile in sequential since unrar Makefile doesn't work in parallel
-    system 'make', '-j1', 'all'
-    system 'make', '-j1', 'lib'
+    system "CPPFLAGS='-fPIC' make -j1 all"
```

Comment:
Is the unrar makefile still unable to compile in parallel?

Reply:
It was failing without the `-fPIC` but I never tried to remove the `-j1` option.